### ISSUE TITLE
TextParserViewHelper produces escaped output

### ISSUE THREAD

**Issue by mgrundkoetter** [DavidBruchmann/typo3_forum#55]
The rendered content contains html tags in my setup. I guess the `protected $escapeOutput = false;` ist missing in the viewhelper.

**Comment by DavidBruchmann** [comment]
I can confirm that there might be some problems, and it's quite possible that I integrated some ugly workarounds because I never knew the parameter `protected $escapeOutput = false;`.

**Comment by mgrundkoetter** [comment]
Yes, thats new in v8 and the default setting is not good in my eyes. Also see the comment on the answer in https://stackoverflow.com/questions/50408495/typo3-own-viewhelper-escapes-html-in-t3-v8 #nodocs 👎

**Comment by DavidBruchmann** [comment]
I'd like to remove my ugly workarounds in the templates with this commit.
You think you could integrate it?

**Comment by mgrundkoetter** [comment]
What exactly was your workaround like?

**Comment by DavidBruchmann** [comment]
I just found one in `Partials/Forum/List.html` it's looking like this:
```
<f:format.html><mmf:forum.forumIcon forum="{forum}" width="32"/></f:format.html>
```

**Comment by DavidBruchmann** [comment]
another on in `Partials/Standard/Post/Single.html`:
```
<f:format.html><mmf:format.textParser content="{post.author.signature}"/></f:format.html>
```
there I also found `<f:comment></f:comment>`, perhaps remove those things too, if you don't mind
